server/src/test/java/com/objectcomputing/checkins/services/TestContainersSuite.java

```diff
@@ -0,0 +1,41 @@
+package com.objectcomputing.checkins.services;
+
+import com.objectcomputing.checkins.services.fixture.RepositoryFixture;
+import io.micronaut.runtime.server.EmbeddedServer;
+import io.micronaut.test.annotation.MicronautTest;
+import org.flywaydb.core.Flyway;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.testcontainers.containers.PostgreSQLContainer;
+import org.testcontainers.junit.jupiter.Container;
+import org.testcontainers.junit.jupiter.Testcontainers;
+import javax.inject.Inject;
+
+@MicronautTest(environments = "test", transactional = false)
+@Testcontainers
+public abstract class TestContainersSuite implements RepositoryFixture {
+
+    @Container
+    private static final PostgreSQLContainer<?> postgres = new PostgreSQLContainer<>("postgres:11.6");
```

Comment:
I wanted to make this hardcoded postgres a property in the application.yml, but my `@Value("${tcimage:postgres:11.6}")` wasn't working. If anyone can figure it, please push to this branch =)

Reply:
Did you try doing something like: https://github.com/oci-labs/check-ins/blob/develop/server/src/main/java/com/objectcomputing/checkins/gcp/postgres/GoogleCloudDatabaseSetup.java#L14

Reply:
I think it has something to do with `@Testcontainers` and `@Container` it looks like it tries to start and create that first before Micronaut has a chance to inject the value from the application.yml. I'll look in to it a little bit.

Reply:
Well I didn't get to this today like I wanted. I'm going to punt on this one for now. I'll fix it later, won't be hard to retrofit in later.